ykrt/src/mt.rs

```diff
@@ -24,10 +27,11 @@ const PHASE_NUM_BITS: usize = 2;
 // accessed without any further operations after the initial tag check.
 const PHASE_TAG: usize = 0b11; // All of the other PHASE_ tags must fit in this.
 const PHASE_COMPILED: usize = 0b00; // Value is a pointer to a chunk of memory containing a
-                                    // compiled trace.
+                                    // CompiledTrace<I>.
 const PHASE_TRACING: usize = 0b01; // Value is a pointer to a malloc'd block that allows us to
                                    // precisely identify when we have reached the same Location.
-const PHASE_COUNTING: usize = 0b10; // The value specifies how many times we've seen this Location.
+const PHASE_COMPILING: usize = 0b10; // Value is a pointer to an `Box<CompilingTrace<I>>`.
```

Comment:
an -> a?

Reply:
Fixed in https://github.com/softdevteam/yk/pull/171/commits/51c3bb80a3bd6ffc143543d45064abd492be8831.